api/receiver/create_run.go

```diff
@@ -22,13 +22,16 @@ import (
 const InternalUsername = "_processor"
 
 // HandleResultsCreate handles the POST requests for creating test runs.
+// nolint:staticcheck // TODO: Fix staticcheck lint error (testRun.Revision).
```

Comment:
The staticcheck errors are related to testRun.Revision which was labelled as deprecated. 

This [pull request](https://github.com/web-platform-tests/wpt.fyi/pull/164) suggests that FullRevisionHash is optional while this [issue](https://github.com/web-platform-tests/wpt.fyi/issues/49) suggests that Revision should only be used as a fallback. Should I fix the errors by using FullRevisionHash?

Reply:
This issue is related to SA1019, I've added the staticcheck information for the relevant issues!